exporter/exporterhelper/metricshelper.go

```diff
@@ -31,6 +31,7 @@ type PushMetricsData func(ctx context.Context, td consumerdata.MetricsData) (dro
 type metricsExporter struct {
 	exporterName    string
 	pushMetricsData PushMetricsData
+	shutdownFunc    Shutdown
```

Comment:
nit: this is not consistent with the naming for `pushMetricsData`: it is also a function but doesn't have the `Func` suffix. I'm fine with either. Anyway the code should be consistent (I bet the overall code base is not, but that is a separate issue, I will open it).

Reply:
I appreciate consistency. I'll fix it.